mmv1/third_party/terraform/utils/iam_dataproc_job.go.erb

```diff
@@ -86,14 +91,17 @@ func DataprocJobIdParseFunc(d *schema.ResourceData, config *Config) error {
 }
 
 func (u *DataprocJobIamUpdater) GetResourceIamPolicy() (*cloudresourcemanager.Policy, error) {
+<% if version == "ga" -%>
 	req := &dataproc.GetIamPolicyRequest{}
+<% end -%>
 
 	userAgent, err := generateUserAgentString(u.d, u.Config.userAgent)
 	if err != nil {
 		return nil, err
 	}
 
-	p, err := u.Config.NewDataprocClient(userAgent).Projects.Regions.Jobs.GetIamPolicy(u.GetResourceId(), req).Do()
+<%# GetIamPolicy signature changes between beta and GA clients %>
```

Comment:
Any idea why this is the case?

Reply:
Requested version info changed from being included in the constructor to a builder pattern. Not sure why this changed, but we see both types across various IAM resources.